frontend/src/scenes/data-warehouse/settings/DataWarehouseSourcesTable.tsx

```diff
@@ -0,0 +1,325 @@
+import { router } from 'kea-router'
+import { ProductIntroduction } from 'lib/components/ProductIntroduction/ProductIntroduction'
+import { More } from 'lib/lemon-ui/LemonButton/More'
+import { urls } from 'scenes/urls'
+
+import { DataTableNode, NodeKind } from '~/queries/schema'
+import { ExternalDataSourceSchema, ExternalDataStripeSource, ProductKey } from '~/types'
+
+import { dataWarehouseSettingsLogic } from './dataWarehouseSettingsLogic'
+
+const StatusTagSetting = {
+    Running: 'primary',
+    Completed: 'success',
+    Error: 'danger',
+    Failed: 'danger',
+}
+
+export function DataWarehouseSourcesTable(): JSX.Element {
+    const { dataWarehouseSources, dataWarehouseSourcesLoading, sourceReloadingById } =
+        useValues(dataWarehouseSettingsLogic)
+    const { deleteSource, reloadSource } = useActions(dataWarehouseSettingsLogic)
+
+    const renderExpandable = (source: ExternalDataStripeSource): JSX.Element => {
+        return (
+            <div className="px-4 py-3">
+                <div className="flex flex-col">
+                    <div className="mt-2">
+                        <SchemaTable schemas={source.schemas} />
+                    </div>
+                </div>
+            </div>
+        )
+    }
+
+    if (!dataWarehouseSourcesLoading && dataWarehouseSources?.results.length === 0) {
+        return (
+            <ProductIntroduction
+                productName="Data Warehouse Source"
+                productKey={ProductKey.DATA_WAREHOUSE}
+                thingName="data source"
+                description="Use data warehouse sources to import data from your external data into PostHog."
+                isEmpty={dataWarehouseSources?.results.length == 0}
+                docsURL="https://posthog.com/docs/data-warehouse"
+                action={() => router.actions.push(urls.pipelineNodeDataWarehouseNew())}
+            />
+        )
+    }
+
+    return (
+        <LemonTable
+            dataSource={dataWarehouseSources?.results ?? []}
+            loading={dataWarehouseSourcesLoading}
+            disableTableWhileLoading={false}
+            columns={[
+                {
+                    title: 'Source Type',
+                    key: 'name',
+                    render: function RenderName(_, source) {
+                        return source.source_type
+                    },
+                },
+                {
+                    title: 'Table Prefix',
+                    key: 'prefix',
+                    render: function RenderPrefix(_, source) {
+                        return source.prefix
+                    },
+                },
+                {
+                    title: 'Status',
+                    key: 'status',
+                    render: function RenderStatus(_, source) {
+                        return <LemonTag type={StatusTagSetting[source.status] || 'default'}>{source.status}</LemonTag>
+                    },
+                },
```

Comment:
I think using `active` | `paused` doesn't make sense for the sources, unfortunately. It would be nice to have it unified, but it's good to know when the sources are running (some are quite long-running).

Reply:
Just to make sure I understand, you essentially have 3 modes:
1. running (when the import is actually running)
2. enabled, but not currently running
3. disabled

And it's important to distinguish between 1 & 2

Reply:
We also have `Error` and `Failed` (although I am not sure of the difference between those two). But yeah, we need to distinguish between running and "enabled"/the last sync completed successfully

Reply:
Question: if there's an error, then it's in a paused state afterwards? Assuming yes, how about we do:
1. `Running` <- we might make batch exports also use that if there's an active run
2. `Active` <- not running, waiting for the scheduled time, similar to batch exports
3. `Error` <- when paused due to failure, we'll have that for batch exports once we start auto-pausing them too
4. `Paused` <- when user selected to pause it, I assume that's an option

For context: Batch exports have a runs page https://github.com/PostHog/posthog/pull/22552 where we show info about historical runs

Reply:
"Paused" takes on a different meaning with data warehouse imports at the moment. Paused will pop up when our row limit's kick in and stop the jobs in progress for a team. We don't have any explicit controls for a user to pause right now. The imports are either running or you cancel while they're running or completely delete. 

I've aligned the states to Running/Active/Error here https://github.com/PostHog/posthog/pull/22580 and I think the follow up on making paused make sense will be done in a follow up when we enable pausing a sync